.github/workflows/build.yml

```diff
@@ -77,8 +84,15 @@ jobs:
         run: echo DESTINY_VERSION=${{ github.head_ref }} >> $GITHUB_ENV
         if: ${{ github.ref_type != 'tag' }}
 
+      # https://docs.flutter.dev/development/tools/sdk/releases?tab=linux
       - name: "Setup flutter"
-        uses: "subosito/flutter-action@v2.4.0"
+        uses: subosito/flutter-action@v2
+        with:
+          # set fixed stable version, to avoid surprises, when flutter upgrades
+          flutter-version: '3.3.10'
```

Comment:
This looks like we want a variable to have that Flutter version only once. Maybe that helps: https://docs.github.com/en/actions/learn-github-actions/variables